pkg/options/common.go

```diff
@@ -12,37 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package maker
+package options
 
 import (
 	"fmt"
 
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 
 	bundle "github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/apis/bundle/v1alpha1"
-	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/filter"
 )
 
-// ObjHandler is a function that can 'make' objects.
-type ObjHandler func(obj *unstructured.Unstructured, ref bundle.ComponentReference, pm ParamMaker) (*unstructured.Unstructured, error)
+// ObjHandler is a function that can apply options to a specific Kubernetes
```

Comment:
Not specific, right?  It'll take any K8s object.

Reply:
Done